DataFormats/TrackReco/interface/TrackBase.h

```diff
@@ -683,6 +710,18 @@ inline const TrackBase::Point & TrackBase::referencePoint() const
     return vertex_;
 }
 
+// Time at the reference point on the track
+inline double TrackBase::t0() const
+{
+    return t0_;
+}
+
+// Velocity at the reference point on the track
+inline double TrackBase::beta() const
+{
+    return beta_;
```

Comment:
Is this an absolute velocity, or is it normalized to c? Please clarify in the comments.